healthtools/scrapers/health_facilities.py

```diff
@@ -28,17 +18,28 @@ def __init__(self):
         self.data_archive_key = "archive/health_facilities-{}.json"
 
         self.access_token = None
+        self.total_pages = 0
 
-    def scrape_site(self):
+    def scrape_site(self, page_no=1):
         self.get_token()
-        self.get_data()
-
+        self.get_data(page_no)
         if self.results:
-            self.archive_data(json.dumps(self.results))
             self.elasticsearch_delete_docs()
```

Comment:
If we scrape the next page and it fails, won't we have lost all the data in elasticsearch or have incomplete data in elasticsearch?